schema.sql

```diff
@@ -35,4 +35,27 @@ DROP COLUMN species;
 
```

Comment:
- [ ] I think it would be better if you update your `Getting Started` section by adding information needed by users in order to `clone` your project locally, `access` the cloned directory, `use` your app,... And I also suggest your remove other empty sections which don't have information instead of writing `No need to install...`,...
This will make your documentation be more descriptive and professional. Example:
```
...
### Setup

1. Clone this repository to your desired folder by running the following commands in your terminal:

  git clone https://github.com/tsheporamantso/math-magicians.git

2. access the cloned directory run:
  cd math-magicians
...
```
![Screenshot from 2023-05-05 07-43-47](https://user-images.githubusercontent.com/83042742/236384945-27dfcac9-e372-4de7-9c78-238437a2a97b.png)